src/main/java/org/zalando/pazuzu/feature/FeaturesResource.java

```diff
@@ -32,10 +32,16 @@ public FeaturesResource(FeatureService featureService) {
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public List<FeatureDto> listFeatures(
             @RequestParam(required = false, name = "name") String[] featureNames,
-            @RequestParam(required = false, name = "sorted") Integer sorting)
+            @RequestParam(required = false, name = "sorted") Integer sorting,
+            @RequestParam(required = false, name = "offset") Integer offset,
+            @RequestParam(required = false, name = "limit") Integer limit)
             throws ServiceException {
         if (featureNames == null) {
-            return featureService.listFeatures("", FeatureDto::ofShort);
+            if (offset != null && limit != null) {
+                return featureService.listFeatures(offset, limit, FeatureDto::ofShort);
+            } else {
+                return featureService.listFeatures("", FeatureDto::ofShort);
```

Comment:
Does it by default load the full list? Maybe we should provide sensible default values in that case?